src/shady-css/tokenizer.js

```diff
@@ -25,10 +26,14 @@ class Tokenizer {
    */
   constructor(cssText) {
     this.cssText = cssText;
-    this.offset = 0;
+    this[cursorToken] = new Token(Token.type.none, 0, 0);
```

Comment:
Can you document the class fields? Given just the names, I'm not sure what the difference between `cursorToken` and `currentToken` would be.